### PR TITLE
Office hours meeting notes (Tue 09 Jul 2019)

### PR DESCRIPTION
**Pull request by alessandrogario**

### CODE REVIEW

File: OFFICE_HOURS/20190709_meeting_notes.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+We may like to use https://opencollective.com/ to do transparent budgeting and accounting, and then use it to solicit funding from stakeholders. We can seek advisement from Linux Foundation about this.
+
+Operational costs of the osquery project include:
+* the osquery Slack channel
+* establishing a legal entity that can register for developer programs at Microsoft, Apple
+* code signing certs and their associated developer programs
+* the CI infrastructure (Azure Pipelines)
+* S3 hosting for cached binaries of pre-built dependencies
+* website hosting (it's currently the no-cost GitHub Pages, but we'll move it back to Netlify in order to restore the ability to share deep links)
```
Comment (groob) [inline OFFICE_HOURS/20190709_meeting_notes.md]: This is not a done decision/it's a discussion. The 404 handler seems to work fine right now. (just clarifying what I said)

Comment (directionless) [reply OFFICE_HOURS/20190709_meeting_notes.md]: Is netlify a requirement for this? It seems like there may be many options

**Comment by directionless** [comment]
I don't like the all caps `OFFICE_HOURS`, but I'm a big 👍 for notes in git.

I'm good with this as is, and we can discuss moving things around later?

**Comment by alessandrogario** [comment]
> I don't like the all caps `OFFICE_HOURS`, but I'm a big 👍 for notes in git.
> 
> I'm good with this as is, and we can discuss moving things around later?

I'll convert the old notes as well! I don't like the all caps either, I just copied the style from the other file names! I can rename it right now, what would you like to use?

**Comment by directionless** [comment]
My gut sense is to use `office-hours/` or `docs/office-hours/`. But I also want to move everything to a hugo site in the future

**Comment by alessandrogario** [comment]
> My gut sense is to use `office-hours/` or `docs/office-hours/`. But I also want to move everything to a hugo site in the future

I moved the file here: `docs/office-hours/20190709_meeting_notes.md`

File: README.md [hunk 2]
```diff
@@ -2,3 +2,4 @@
 
 * [osquery Charter v1.0](CHARTER.md)
 * [osquery Code of Conduct v1.0](CODE_OF_CONDUCT.md)
+* [Office hours meeting notes](OFFICE_HOURS)
```
Comment (directionless) [inline README.md]:
```suggestion
* [Office hours meeting notes](/docs/office-hours/)
```

File: OFFICE_HOURS/20190709_meeting_notes.md (hunk 1 above)
Comment (groob) [reply OFFICE_HOURS/20190709_meeting_notes.md]: You know me. I’m down with hosting SPAs on GCP.

File: docs/office-hours/20190709_meeting_notes.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+## Chocolatey support
+
+The osquery package on Chocolatey is still owned by Facebook, and we are not able to update it.
+
+Supporting the ability to build MSI installers for osquery is a higher priority than supporting the Chocolatey package repository, but we would like to support both.
+
+# Funding
+
+## Open Collective
```
Comment (caniszczyk) [inline docs/office-hours/20190709_meeting_notes.md]: FYI the LF has multiple ways of you raising money for your project in a neutral way, either through membership dues or via its https://communitybridge.org platform. We should sync up with what you want to do with raising money, but the individual donation approach doesn't work well, usually a dominant membership driven approach like CNCF or a mixed approached like letsencrypt.org that is hosted by the LF too works well.

Anyways, lets have a conversation about the project infra/costs needs and we can help.

File: docs/office-hours/20190709_meeting_notes.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+We may like to use https://opencollective.com/ to do transparent budgeting and accounting, and then use it to solicit funding from stakeholders. We can seek advisement from Linux Foundation about this.
+
+Operational costs of the osquery project include:
+* the osquery Slack channel
+* establishing a legal entity that can register for developer programs at Microsoft, Apple
+* code signing certs and their associated developer programs
+* the CI infrastructure (Azure Pipelines)
+* S3 hosting for cached binaries of pre-built dependencies
+* website hosting (it's currently the no-cost GitHub Pages, but we'll move it back to Netlify in order to restore the ability to share deep links)
```
Comment (caniszczyk) [inline docs/office-hours/20190709_meeting_notes.md]: LF can help here, we have a deal with netlify

File: OFFICE_HOURS/20190709_meeting_notes.md (hunk 1 above)
Comment (zwass) [reply OFFICE_HOURS/20190709_meeting_notes.md]: I fixed the deep-linking issue in https://github.com/osquery/osquery-site/pull/138. You should be able to load https://osquery.io/schema/3.3.2#arp_cache.

Is there a separate issue that needs addressing?

I am a fan of netlify, and I think a free plan would do us just fine. I'm not sure we have any sever deficiencies with the Github hosting currently, so I wouldn't say this is a priority.

**Comment by alessandrogario** [comment]
@caniszczyk && @groob you can maybe add your own notes to the document, so we can use them for the next agenda!

**Pull request merged**